recipes/tritimap/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = "0.9.0" %}
+{% set sha256 = "c3a0249f4622fe7b1f4d2d6b716dfb0dd5a9bd6414ac2669ebc3239d50855c58" %}
+
+
+package:
+  name: tritimap
+  version: {{ version }}
+
+source:
+  url: https://github.com/fei0810/Triti-Map/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - tritimap=tritimap.tritimap:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python =3.9.2
+    - click >=7
+    - snakemake >=6.0.3
+    - biopython
+    - fastp >=0.20.1
+    - bioawk
+    - bwa-mem2 =2.2.1
+    - samtools =1.12
+    - seqkit >=0.15.0
+    - spades =3.15.0
+    - star =2.7.6a
+    - abyss =2.0.2
+    - bedops =2.4.39
+    - bwa =0.7.17
+    - gatk4 >=4.2.0
+    - hmmer =3.3.2
+    - minimap2 >=2.17
+    - xmltramp2 =3.1.1
+    - r-base =4.0.2
+    - r-jsonlite =1.7.2
+    - r-qtlseqr =0.7.5.2
+
+test:
+  commands:
+    - tritimap --help
+    - tritimap --version
+
+about:
+  home: https://github.com/fei0810/Triti-Map
+  dev_url: https://github.com/fei0810/Triti-Map
+  doc_url: https://github.com/fei0810/Triti-Map/wiki
+  license: MIT
+  license_file: LICENSE
+  summary: Triti-Map is a Snakemake-based pipeline for gene mapping in Triticeae.
+  description: |
+    Triti-Map is a Snakemake-based pipeline for gene mapping in Triticeae, which contains a suite of user-friendly computational packages and web-interface integrating multi-omics data from Triticeae species including genomic, epigenomic, evolutionary and homologous information.
+    
+    Triti-Map could efficiently explore trait-related genes or functional elements not present in the reference genome and reduce the time and labor required for gene mapping in large genome species.
+
+extra:
+  recipe-maintainers:
+    - Fei Zhao
+    - Zhougeng Xu
```

Comment:
Can you make these github ids (e.g., fei0810)? Then you'll be notified to approve automatically generated update PRs.

Reply:
Hi, Ryan. I have changed the recipe-maintainers IDs.